Makefile

```diff
@@ -44,6 +44,7 @@ endif
 
 # Make sure we are in the same directory as the Makefile
 BASE_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+PARENT_DIR := $(dir $(abspath $(BASE_DIR)/..))
```

Comment:
PARENT_DIR is set but not used later. Probably for DOCKER_BUILDROOT ?

Reply:
Removed in latest commit.